allennlp/tests/models/language_model_test.py

```diff
@@ -44,7 +46,6 @@ def test_forward_pass_runs_correctly(self):
                                            decimal=3)
         else:
             np.testing.assert_almost_equal(loss, forward_loss, decimal=3)
-            assert result["backward_loss"] is None
```

Comment:
How about we add a test case here that calls `forward_on_instances` and verifies that it doesn't crash? This matches the bug that you're fixing with this PR and I believe will address the PR test coverage requirement. In general it's good practice to add a test when you fix a bug to prevent regressions. It also makes it clear to people reading the code that the bug was really addressed.